src/15utility.js

```diff
@@ -339,7 +343,14 @@ var loadFile = utils.loadFile = function(path, asy, success, error) {
                 }
             }
         }
-        //*/

```

Comment:
The end of the comment is removed (used to strip out things never used for the browser) - please add again (after the code you added)